web/ui/src/Pools/PoolDetailsController.js

```diff
@@ -159,11 +163,21 @@
                 }
             });
         };
-        
+
+        function setCurrentPool(pool){
+            $scope.currentPool = new Pool(pool);
+            $scope.currentPool.fetchHosts();
+        }
+
+        // update/refresh current pools data
+        function update(){
+            // TODO - use v2 endpoint once its in
```

Comment:
This call to the v2 api endpoint (using the UI's v1 api object) will be replaced with the correct call in the upcoming commits for the 101 pools UI changes.